index.html

```diff
@@ -15,35 +15,28 @@
   <body>
     <div class="main-content">
       <header id="top">
-        <div id="nav_bar">
+        <nav>
           <h1 class="logo"><a href="#top">CED<sup>music</sup></a></h1>
-          <nav>
             <ul>
-              
               <li><a href="#au">About Us</a></li>
               <li><a href="#serv">Our Services</a></li>
               <li><a href="#team_meet">Meet the Team</a></li>
               <li><a href="#contact">Contact Us</a></li>
             </ul>
           </nav>
-        </div>
       </header>
       <main>
         <banner>
           <div class="title_container">
-            <div>
-              <h1 id="title">CED Music</h1>
-            </div>
-            <div>
-              <span id="caption">The place for all your music, production, and sound system needs.</span>
-            </div>
```

Comment:
Generally it's better to use as few divs and spans as possible in HTML - they're completely generic elements so they don't give any information about what's within them. Using e.g. the banner element (as you have here!) is much better (same with section elements below) because it means the HTML is more self describing and easier to understand for devs, browsers and screen readers. Likewise, having these little divs that wrap up individual items of text is unnecessary.

Here's an article that goes into the idea of Semantic HTML in a bit more detail. https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i